jobboard/models.py

```diff
@@ -72,15 +81,18 @@ def get_jobs_by_job_type(cls, job_type):
         return set(cls.objects.filter(job_type=job_type))
 
     @classmethod
-    def get_jobs_by_keywords(cls, *keywords_as_string):
+    def get_jobs_by_keywords(cls, keywords_as_string):
         """
-        Allows students to search for a job based on one ore more keywords linked to the job
+        Allows students to search for a job based on one ore more keywords linked to the job.
```

Comment:
or* :)

Reply:
and - why did you deleted the * ?
Wasn't it there so we could send this fun multiple keywords?

Reply:
Because this function is called with the input entered in the search bar therefore the argument won't be an iterable. I use the .split() function on the argument so that if the user enters more than one keywords it puts them in a list and then I can still search for jobs based on each keyword.